### PR TITLE
Comments proceeding

### PR DESCRIPTION
**Pull request by joleroi**
Hi @cdeil 

Here are my comments for the proceeding. Uncontroversial stuff is done in this PR, I tried to keep the diff clean, so you should be able to check the changes if you wish. Furthermore, I have the following comments

* There is a lot of emphasis on unbinned fitting in the Introduction section. I would spend max 1 sentence on this. If you want to make the point, that we can implement new stuff, it should not be made in the introduction
* Many URLs go over the text body. You could try``\PassOptionsToPackage{hyphens}{url}\usepackage{hyperref}``, see [here](https://tex.stackexchange.com/questions/3033/forcing-linebreaks-in-url)
* As always, I would not spend much time on describing Gammalib/CTOOLs. Just mention that it's there.

### CODE REVIEW

File: proceeding/gammapy-icrc2017.tex [hunk 1]
```diff
@@ -91,7 +91,7 @@ \section{Introduction}
 \label{sec:intro}
 
 % CTA science tools
-The Cherenkov Telescope Array (CTA) will observe the sky in very-high-energy
+The Cherenkov Telescope Array (CTA) will observe the sky in very high energy (VHE, E > 100$\,$GeV)
```
Comment (cdeil) [inline proceeding/gammapy-icrc2017.tex]: I think the CTA statement is that they will observe at much lower energy, I don't think we can put 100 GeV here.

**Review by cdeil — CHANGES_REQUESTED** [review]
Thanks! I've left one inline comment. And could you please remove the update to `proceeding/gammapy-icrc2017.pdf` here?

And if if you have time, if you could to the update so that the URLs get properly formatted, that would be much appreciated. Currently we have one URL with an underscore and I didn't manage to format it at all so that it becomes a link. Would the suggestion you made to avoid the wrapping work for that as well?

I've gotten a few other comments and will make a new version tonight or tomorrow morning.

File: proceeding/gammapy-icrc2017.tex (hunk 1 above)
Comment (cdeil) [reply proceeding/gammapy-icrc2017.tex]: I think "very-high-energy" is like "gamma-ray", i.e. use dashes if used as an adjective like here, no?

**Comment by cdeil** [comment]
> As always, I would not spend much time on describing Gammalib/CTOOLs. Just mention that it's there.

Agreed that it should be rather shortened than extended. But I thought there should be some statement how it's different. If you have a concrete suggestion how to shorten, please make a suggestion in a separate commit. (If not, I'll have a look later and try to shorten myself).

File: proceeding/gammapy-icrc2017.tex (hunk 1 above)
Comment (cdeil) [reply proceeding/gammapy-icrc2017.tex]: Catherine commented: should be VHE, E > 20 GeV in CTA statements.
@joleroi - Can you update please?
(or should I take over)

**Comment by joleroi** [comment]
I added some linebreak in the document manually. I cannot find the place in the document where the packages are included, so it's hard to add any option to them. Grepping for ``usepackage`` did not yield any result.

**Comment by cdeil** [comment]
At the moment we're just using `\documentclass{PoS}`
https://github.com/gammapy/icrc2017-gammapy-poster/blob/master/proceeding/gammapy-icrc2017.tex#L1

You could add `\usepackage` below that line if you need something.
I don't know if that would overwrite options for previous config that we drag in from `\documentclass{PoS}`.
Maybe just try?

**Comment by cdeil** [comment]
Merging this now, thanks!

I'll try out your suggestion on URL formatting in master.

**Pull request merged**